frontend/awx/resources/projects/hooks/useProjectActions.tsx

```diff
@@ -26,10 +35,14 @@ export function useProjectActions(options: { onProjectsDeleted: (projects: Proje
       project?.summary_fields?.user_capabilities?.edit
         ? ''
         : t(`The project cannot be edited due to insufficient permission`);
-    const cannotSyncProject = (project: Project) =>
-      project?.summary_fields?.user_capabilities?.start
-        ? ''
-        : t(`The project cannot be synced due to insufficient permission`);
+    const cannotSyncProject = (project: Project) => {
+      if (project.scm_type === '') {
+        return t(`Cannot sync project`);
+      } else if (project.scm_type !== '' && !project?.summary_fields?.user_capabilities?.start) {
+        return t(`The project cannot be synced due to insufficient permission`);
+      }
+      return '';
+    };
```

Comment:
@kialam I see this [commit](https://github.com/ansible/ansible-ui/pull/357/commits/ed19039f5abf91b7b4bec76c27708117e7923768) in the commit history for disabling the sync when it is progress but I suspect some following commit is undoing that change 🤔 because it's not in the final set of file changes and the Sync button remains enabled when a sync is in progress.

Reply:
I saw that too. The last commit is the offender here - as to how it snuck in ... 🐉 👀 

This also exposed a vulnerability that needs to be patched in my unit tests (I added a test to see if project sync is disabled). I'll have changes pushed on Monday when I'm back from PTO. Thanks!